src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -21,6 +21,19 @@ $(document).ready(function() {
     hideUncommonPanels();
 });
 
+function addLoadingIndicator(button) {
+    currentText = button.text();
+    button.prop('disabled', true);
+    button.append("<img src='/images/ajax-loader.gif'/>");
```

Comment:
Two issues here:
- Switch the `'` and `"`; we use `'` in JS and `"` in HTML.
- Don't close the tag; we don't close void tags based on our style guide.